src/components/Reviews.jsx

```diff
@@ -1,28 +1,39 @@
 import { useEffect } from "react";
 import { getReviews } from "../requests";
 
-const Reviews = ({ reviews, setReviews }) => {
+const Reviews = ({ reviews, setReviews, loading, setLoading }) => {
   useEffect(() => {
     getReviews().then((data) => {
       setReviews(data);
+      setLoading(false);
     });
   }, []);
-  return (
-    <section>
-      <ul>
-        {reviews.map((review) => {
-          return (
-            <li key={review.review_id}>
-              <p>{review.title}</p>
-              <p>{review.owner}</p>
-              <p>{review.category}</p>
-              <img id="review-image" src={review.review_img_url}></img>
-            </li>
-          );
-        })}
-      </ul>
-    </section>
-  );
+
+  if (loading === true) {
+    return <section>Loading...</section>;
+  }
+  if (loading === false) {
+    return (
+      <section>
+        <ul>
+          {reviews.map((review) => {
+            return (
+              <li key={review.review_id}>
+                <p>{review.title}</p>
+                <p>{review.owner}</p>
+                <p>{review.category}</p>
+                <img
+                  id="review-image"
+                  src={review.review_img_url}
+                  alt="no image available"
```

Comment:
no image available isn't super helpful.  Use some property that exists on the review.  What is the image meant to be portraying?